patches/server/0468-Implement-Chunk-Priority-Urgency-System-for-Chunks.patch

```diff
@@ -117,38 +79,31 @@ index d907872d80f840b343419f49a6708082da6f921b..ce320672d7602c94dd75ad857435dca6
 +import it.unimi.dsi.fastutil.longs.Long2ObjectOpenHashMap; // Paper
```

Comment:
no longer needed

Reply:
that import is need for L121 public final Long2ObjectOpenHashMap<Integer> neighborPriorities = new Long2ObjectOpenHashMap<>(); could inline it

Reply:
yeah that's what I mean when I say that, this can all be inlined now since there is no `it` class